INSTALL

```diff
@@ -17,13 +17,23 @@ Getting Subsurface source
 -------------------------
 
 You can get the sources to the latest development version from our git
-repository http://github.com/Subsurface-divelog/subsurface.git
-
-Remark, if you plan on making pull requests, the best way is to clone
-subsurface.git to your GitHub account, make a new branch (e.g. myPatches).
-Then clone that to your local computer. Doing this makes it very simple
-to generate pull requests.
-
+repository:
+    git clone http://github.com/Subsurface-divelog/subsurface.git
+
+You keep it updated by doing:
+    git checkout master
+    git pull -r
+    
+If you plan on making pull requests, you need furthermore to:
+   - clone subsurface.git to your GitHub account
+   - make a new local work branch, name it after the patch like e.g. myFirstPatch
```

Comment:
That's actually optional. It's highly recommended, but not required. You could work in your master and create the pull request from there. So maybe line 27 above should read "Here are some best practices for those planning to contribute and create pull requests for Subsurface:"
And furthermore - why is this section in the INSTALL file? Isn't this part of CONTRIBUTING.md?
So I'd suggest removing 27-36 from INSTALL